packages/graph/hash_graph/lib/graph/src/knowledge/link/mod.rs

```diff
@@ -0,0 +1,114 @@
+use std::{collections::HashMap, fmt};
+
+use serde::{Deserialize, Serialize};
+use utoipa::Component;
+
+use super::EntityId;
+use crate::ontology::types::uri::VersionedUri;
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Component)]
+pub struct Link {
```

Comment:
Naming of the structs/enums in this module has been a little difficult. I settled on this being the "link" itself, though